logging.go

```diff
@@ -196,6 +223,9 @@ func StringToLevel(level string) Level {
 
 // SetLogStderr sets flag for logging stderr output
 func SetLogStderr(enable bool) {
+	if !enable && !isFileLoggingEnabled() {
```

Comment:
why do we want to tie between the two ?

if none are set -> no logging occurs

makes sense ?

Reply:
What this does is:  When file logging is disabled, and the user disabled error logging, it prints an error message to notify the user that no logging will occur:
```
"cni-log: filename is required when logging to stderr is off - will not log anything\n"
```

See unit test:
```
   80             When("error logging is disabled while file logging is disabled", func() {                                   
   81                 It("does report an error", func() {                                                                     
   82                     errStr := captureStdErr(SetLogFile, "")                                                             
   83                     Expect(errStr).To(BeEmpty())                                                                        
   84                     errStr = captureStdErr(SetLogStderr, false)                                                         
   85                     Expect(errStr).To(ContainSubstring(logFileReqFailMsg))                                              
   86                 })                                                                                                      
   87             })    
```

If such an error message is pointless, I can remove it

Reply:
It should log once stating both are disabled, then no longer log.
I am okay with this approach